src/NServiceBus.Persistence.DynamoDB.PersistenceTests/When_failing_to_acquire_lock.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace NServiceBus.PersistenceTesting
+{
+    using System.Threading.Tasks;
+    using System;
+    using System.Threading;
+    using NUnit.Framework;
+    using Sagas;
+
+    public class When_failing_to_acquire_lock : SagaPersisterTests
```

Comment:
add a comment that explains the similarity to `When_concurrent_update_exceed_lock_request_timeout_pessimistic`